Cargo.toml

```diff
@@ -129,8 +137,10 @@ alloy-transport-ws = { version = "0.1", default-features = false }
 alloy-transport-ipc = { version = "0.1", default-features = false }
 alloy-pubsub = { version = "0.1", default-features = false }
 alloy-json-rpc = { version = "0.1", default-features = false }
-alloy-rpc-client = { version = "0.1", default-features = false }
+alloy-rpc-client = "0.2"
```

Comment:
we should use version 0.1

Reply:
it works if we use the version "0.1" but we have to set `default-features = true`